settings.gradle

```diff
@@ -3,22 +3,22 @@ rootProject.name = 'git-machete-intellij-plugin'
 include 'binding'
 include 'qual'
 include 'testCommon'
-include 'gitCoreApi'
-include 'gitCoreJGit'
-include 'branchLayoutApi'
-include 'branchLayoutImpl'
-include 'backendApi'
-include 'backendImpl'
-include 'frontendResourceBundles'
-include 'frontendCompatibilityBridge'
-include 'frontendDefs'
-include 'frontendDataKeys'
-include 'frontendIcons'
-include 'frontendVfsUtils'
+include 'gitCore:api'
+include 'gitCore:jGit'
+include 'branchLayout:api'
+include 'branchLayout:impl'
+include 'backend:api'
+include 'backend:impl'
+include 'frontend:resourceBundles'
+include 'frontend:compatibilityBridge'
+include 'frontend:defs'
+include 'frontend:dataKeys'
+include 'frontend:icons'
+include 'frontend:vfsUtils'
 include 'frontendActions'
 include 'frontendFile'
```

Comment:
Also! is #796 to be done on this PR as well?

Reply:
I think it can go into a subsequent PR in fact

Reply:
yes

Reply:
Unlinking then...